### PR TITLE
Nano modifiers in 112X

### PR DESCRIPTION
**Pull request by mariadalfonso**
backport of https://github.com/cms-sw/cmssw/pull/32463

### CODE REVIEW

**Comment by mariadalfonso** [comment]
please test workflow 136.8523

**Comment by mariadalfonso** [comment]
+xpog

**Comment by chayanit** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -459,6 +459,9 @@
 steps['RunDisplacedJet2018C']={'INPUT':InputInfo(dataSet='/DisplacedJet/Run2018C-v1/RAW',label='2018C',events=100000,location='STD', ls=Run2018C)}
 steps['RunCharmonium2018C']={'INPUT':InputInfo(dataSet='/Charmonium/Run2018C-v1/RAW',label='2018C',events=100000,location='STD', ls=Run2018C)}
 
+Run2018Cpilot={319348: [[1, 100]]}
+steps['RunJetHT2018C_nanoULremini']={'INPUT':InputInfo(dataSet='/JetHT/Run2018C-UL2018_MiniAODv2_pilot1-v1/MINIAOD',label='2018C',events=100000,location='STD', ls=Run2018Cpilot)}
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_steps.py:463]: Why do we need to use pilot dataset here? Pilot dataset should be removed anyway when processing is done. Please move to real Legacy dataset, and contact PdmV to lock specific block at FNAL or CERN where relvals will run.

Comment (srimanob) [reply Configuration/PyReleaseValidation/python/relval_steps.py:463]: FYI @chayanit

Comment (wajidalikhan) [reply Configuration/PyReleaseValidation/python/relval_steps.py:463]: Hi, 

Here is the realy Legay dataset which is still under production but can be used:
/JetHT/Run2018C-UL2018_MiniAODv2-v1/MINIAOD

https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset+dataset%3D%2FJetHT%2FRun2018C-UL2018_MiniAODv2*%2FMINIAOD+status%3DPRODUCTION

**Comment by wajidalikhan** [comment]
Hi,

Here is the realy Legay dataset which is still under production but can be used:
/JetHT/Run2018C-UL2018_MiniAODv2-v1/MINIAOD

https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset+dataset%3D%2FJetHT%2FRun2018C-UL2018_MiniAODv2*%2FMINIAOD+status%3DPRODUCTION

**Comment by srimanob** [comment]
Please test

**Comment by mariadalfonso** [comment]
+xpog

**Comment by srimanob** [comment]
+Upgrade

**Comment by chayanit** [comment]
+1

**Comment by silviodonato** [comment]
@mariadalfonso could you propagate https://github.com/cms-sw/cmssw/pull/32618/commits/793827be2d313715db03e36e38f2ff7332ab79db to master?

**Comment by silviodonato** [comment]
+operations

**Comment by srimanob** [comment]
> @mariadalfonso could you propagate [793827b](https://github.com/cms-sw/cmssw/commit/793827be2d313715db03e36e38f2ff7332ab79db) to master?

Hi @silviodonato 
It is merged to master already 
https://github.com/cms-sw/cmssw/pull/32648

**Comment by silviodonato** [comment]
merge

**Comment by santocch** [comment]
+1

**Pull request merged**